ray-operator/controllers/ray/common/pod.go

```diff
@@ -654,6 +666,12 @@ func setContainerEnvVars(pod *corev1.Pod, rayNodeType rayv1.RayNodeType, rayStar
 		}
 		container.Env = append(container.Env, extraTagsEnv)
 	}
+	if !utils.EnvVarExists(utils.REDIS_USERNAME, container.Env) {
+		// setting the REDIS_USERNAME env var from the params
+		if value, ok := rayStartParams["redis-username"]; ok {
+			container.Env = append(container.Env, corev1.EnvVar{Name: utils.REDIS_USERNAME, Value: value})
+		}
+	}
```

Comment:
Although the support for Redis usernames in the Ray core has been merged but hasn't been released yet, I think we can add related code in KubeRay 1.3.